changelog.json

```diff
@@ -1,4 +1,33 @@
 [
+  {
+    "id": 6,
+    "slug": "dynamic-pipeline-support-across-cloud-orchestrators",
+    "title": "Dynamic Pipeline Support Across Cloud Orchestrators",
+    "description": "ZenML now supports dynamic pipelines on AWS Sagemaker and Google Cloud Vertex orchestrators, enabling faster execution and more flexible workflow patterns. Dynamic pipelines can now be run from snapshots with configurable parameters, and include improvements to Kubernetes orchestrator handling and step mapping operations that return future objects with an `unpack()` method for better control flow.",
+    "published_at": "2025-12-02T06:42:19Z",
+    "published": true,
+    "audience": "oss",
+    "labels": [
+      "bugfix",
+      "improvement"
+    ],
+    "docs_url": "https://docs.zenml.io/concepts/steps_and_pipelines/dynamic_pipelines",
+    "should_highlight": false
+  },
+  {
+    "id": 5,
+    "slug": "enhanced-cloud-infrastructure-configuration",
+    "title": "Enhanced Cloud Infrastructure Configuration",
+    "description": "Added regional location support to the GCP Image Builder, allowing users to specify Cloud Build regions for compliance and performance requirements. Improved Docker build options with better typing support for more reliable container image configuration across the SDK and CLI.",
+    "published_at": "2025-12-02T06:42:19Z",
+    "published": true,
+    "audience": "oss",
+    "labels": [
+      "improvement"
+    ],
+    "docs_url": "https://docs.zenml.io/stacks/stack-components/image-builders/gcp#how-to-use-it",
+    "should_highlight": false
+  },
```

Comment:
on second thought maybe here should add rather "dynamic pipelines" as a general feature and remove this can propose wording later